Android/app/src/main/java/app/intra/sys/firebase/AnalyticsWrapper.java

```diff
@@ -102,8 +103,8 @@ private NetworkTypes getNetworkType() {
   }
 
   private void log(Events e, @NonNull BundleBuilder b) {
-    String deviceCountry = countryCode.getDeviceCountry().toUpperCase();
-    String networkCountry = countryCode.getNetworkCountry().toUpperCase();
+    String deviceCountry = countryCode.getDeviceCountry().toUpperCase(Locale.ENGLISH);
```

Comment:
Can you try "Locale.ROOT"?  That seems to be the one that provides standard string formatting behavior.

Reply:
Done